readme.md

```
@@ -55,6 +55,8 @@ VidTwit is a full-featured backend and frontend for a YouTube-style video sharin
 - [Tech Stack](#tech-stack)
 - [Environment Variables](#environment-variables)
 - [Run Locally](#run-locally)
+  - [Docker Setup (Recommended)](#docker-setup-recommended)
+  - [Manual Setup](#manual-setup)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)